deno/rest/v10/interactions.ts

```diff
@@ -37,8 +38,11 @@ type RESTPostAPIBaseApplicationCommandsJSONBody = AddUndefinedToPossiblyUndefine
 		| 'guild_id'
 		| 'name_localized'
 		| 'description_localized'
+		| 'default_member_permissions'
 	>
->;
+> & {
```

Comment:
Should use Partial<Pick<>> instead, and be in the AddUndefinedToPossiblyUndefinedTypes so | undefined gets appended to it

Also ensure dm_permissions works too